graphql/types/OrderForm.graphql

```diff
@@ -1,4 +1,4 @@
-type OrderForm {
+type OrderForm @context(scope: "private") {
```

Comment:
shoud be `@context(scope: PRIVATE)` in schema

Reply:
But... this is what we did in the `admin-graphql`. So it's wrong there?